src/libraries/System.Collections.Immutable/tests/ImmutableArrayBuilderTest.cs

```diff
@@ -880,6 +880,137 @@ public void MoveToImmutableAddRemoveAddToCapacity()
             Assert.Equal(new[] { 0, 1, 2 }, builder.MoveToImmutable());
         }
 
+        [Fact]
+        public void DrainToImmutableEmptyZeroCapacity()
+        {
+            var builder = ImmutableArray.CreateBuilder<int>(0);
```

Comment:
Literally every use of `var` added in this file in this change is against style guidelines, and should (have) be(en) expanded to the target type.

Reply:
Yes... but as with Microsoft.Extensions, the whole project started off with a team with different coding guidelines, and we haven't gone through and made them all conform to ours, at which point we've generally opted for new code in the file to conform to the style used by the rest of the file.

Reply:
Thanks, and yeah, I did intentionally follow the patterns in the file/project per our [contributing dos and don'ts](https://github.com/dotnet/runtime/blob/main/CONTRIBUTING.md#dos-and-donts).

> **DO** follow our coding style (C# code-specific).
> 
> **DO** give priority to the current style of the project or file you're changing even if it diverges from the general guidelines.